Airports/EGXH/Runway.txt

```diff
@@ -1 +1 @@
-08  26  081 261 N052.20.28.180 E000.45.19.190 N052.20.39.980 E000.47.24.930
+08 26 081 261 N052.20.27.440 E000.45.11.210 N052.20.40.880 E000.47.34.540
```

Comment:
```suggestion
08  26  081 261 N052.20.27.440 E000.45.11.210 N052.20.40.880 E000.47.34.540
```
As in the first note on https://github.com/VATSIM-UK/UK-Sector-File/wiki/Runway.txt "Where a runway designator is only numeric (i.e. no L, R or C), an extra space should be added after it for alignment purposes."